### PR TITLE
completed tasks

### PR DESCRIPTION
**Pull request by KatrinGratilova**

### CODE REVIEW

File: src/main/java/org/katrin/JewelsCalculator.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
+package org.katrin;
+
+public class JewelsCalculator {
+    // 0 ms
```
Comment (mkharkhuta) [inline src/main/java/org/katrin/JewelsCalculator.java:4]: 0 ms for what length of jewels?

File: src/test/java/org/katrin/JewelsCalculatorTest.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    }
+
+    @Test
+    public void numJewelsInStonesTwoLoopsTest() {
+        String jewels = "aA";
+        String stones = "aAAbbbb";
+        int expected = 3;
+        int actual = calc.numJewelsInStonesTwoLoops(jewels, stones);
+        Assert.assertEquals(expected, actual);
+    }
```
Comment (mkharkhuta) [inline src/test/java/org/katrin/JewelsCalculatorTest.java:52]: this is a good approach for testing positives, negatives, and some edge cases. for example calculateJewels_JewelsIsEmpty_BadRequestIsThrown() {... }. Keep calling the Given-When-Then pattern. It would be clear what is being tested and what the expected result is

**Review by mkharkhuta — COMMENTED** [review]
Good job, but please, in this task focus more on covering the logic with different tests

**Review by mkharkhuta — APPROVED** [review]
Good

**Comment by KatrinGratilova** [comment]
> Good
Thank you very much! I will be more attentive
